src/taipy/core/pipeline/_pipeline_manager.py

```diff
@@ -93,9 +93,18 @@ def _get_or_create(cls, pipeline_config: PipelineConfig, scenario_id: Optional[S
             owner_id,
             {scenario_id} if scenario_id else None,
         )
+        for task in tasks:
```

Comment:
I think we can refactor this and turn it into a shared function in the _Manager class, something like ```__set_parent_ids_for_children_entities(child_entities: List[Entity]): ...``` and set it to NotImplemented for Scenario, Cycle Manager. What do you think? @jrobinAV

Reply:
I don't think we will benefit from this. We will lose some readability. No? What would we gain?